packages/cache-core/__tests__/cacheHandler.spec.ts

```diff
@@ -228,11 +228,12 @@ describe('CacheHandler', () => {
 
   it('should skip reading / writing data if route listed as no cache', async () => {
     Cache.setCacheStrategy(new FileSystemCache())
-    Cache.addNoCacheRoute(mockCacheKey)
+    Cache.addNoCacheMatchers(mockCacheKey)
```

Comment:
let's add more test cases with regex patterns in such case

Reply:
positive and negative cases

Reply:
done